models/models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+    return db.query(`SELECT * FROM topics`)
+        .then(({ rows }) => {
+            if (rows.length === 0) {
+                // Custom error if no topics are found
+                return Promise.reject({
+                    status: 404,
+                    msg: "No topics found"
```

Comment:
We don't need this custom error handler. We know that the topics have been seeded into the database initially, so this will never be triggered.